src/Dialogs/PropertiesWindow.vala

```diff
@@ -981,23 +976,29 @@ public class Files.View.PropertiesWindow : AbstractPropertiesDialog {
             l_perm.halign = Gtk.Align.START;
             l_perm.use_markup = true;
 
+            var permission_box = new Gtk.ListBox () {
+                selection_mode = NONE
+            };
+            // FIXME: use constants in Gtk4
+            permission_box.get_style_context ().add_class ("frame");
+            permission_box.get_style_context ().add_class ("rich-list");
+            permission_box.get_style_context ().add_class ("boxed-list");
+            permission_box.get_style_context ().add_class ("separators");
+            permission_box.add (perm_button_user);
+            permission_box.add (perm_button_group);
+            permission_box.add (perm_button_other);
+
             perm_grid = new Gtk.Grid () {
                 column_spacing = 6,
-                row_spacing = 6,
-                halign = Gtk.Align.CENTER
+                row_spacing = 6
             };
-            perm_grid.attach (owner_user_label, 0, 1, 1, 1);
-            perm_grid.attach (owner_user_choice, 1, 1, 2, 1);
-            perm_grid.attach (group_combo_label, 0, 2, 1, 1);
-            perm_grid.attach (group_combo, 1, 2, 2, 1);
-            perm_grid.attach (owner_label, 0, 3, 1, 1);
-            perm_grid.attach (perm_button_user, 1, 3, 2, 1);
-            perm_grid.attach (group_label, 0, 4, 1, 1);
-            perm_grid.attach (perm_button_group, 1, 4, 2, 1);
-            perm_grid.attach (other_label, 0, 5, 1, 1);
-            perm_grid.attach (perm_button_other, 1, 5, 2, 1);
-            perm_grid.attach (l_perm, 1, 6, 1, 1);
-            perm_grid.attach (perm_code, 2, 6, 1, 1);
+            perm_grid.attach (owner_user_label, 0, 0);
+            perm_grid.attach (owner_user_choice, 1, 0);
+            perm_grid.attach (group_combo_label, 0, 1);
+            perm_grid.attach (group_combo, 1, 1);
+            perm_grid.attach (l_perm, 0, 2);
+            perm_grid.attach (perm_code, 1, 2);
+            perm_grid.attach (permission_box, 0, 3, 2);
```

Comment:
Is there a reason to alter the order of these elements? I think most users would use the buttons.  The permission code and value are kind of extra information and seems out of place in the middle.